tests/director/tests/application_api_test.go

```diff
@@ -1580,6 +1580,16 @@ func TestMergeApplications(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, newFormation, formation.Name)
 
+	defer func() {
+		t.Logf("Cleaning up formation: %s", newFormation)
+
+		var response graphql.Formation
+		deleteFormationReq := fixtures.FixDeleteFormationRequest(newFormation)
+		err = testctx.Tc.RunOperation(ctx, certSecuredGraphQLClient, deleteFormationReq, &response)
```

Comment:
Add error check for the request execution - `require.NoError(t, err)` and you could also assert the formation name - `require.Equal(newFormation, response.Name)`

Reply:
done